docs/modules/ROOT/partials/storage-ceph-remove-mon.adoc

```diff
@@ -50,29 +50,42 @@ kubectl --as=cluster-admin -n syn-rook-ceph-cluster patch cephcluster cluster --
   }]'
 ----
 
-. Cordon node(s) to {mon-operation}
-+
-[source,bash,subs="attributes+"]
-----
-for node in $(echo -n {mon-replace-list}); do
-  kubectl --as=cluster-admin cordon "${node}"
-done
-----
-+
 . For every id in `$MON_IDS` replace the MON pod
 +
-[source,bash]
+[source,bash,subs="attributes+"]
 ----
 mon_id=<MON_ID>
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pod \
```

Comment:
Why didn't this work (anymore)? I'm not happy with replacing a simple step which should trigger the rook operator to do the right thing with a fully manual step that can cause massive chaos.